staging_service/app.py

```diff
@@ -1,107 +1,130 @@
+import json
 import os
 import shutil
+from pprint import pprint
+from urllib.parse import parse_qs
 
 import aiohttp_cors
 from aiohttp import web
 
+from .AutoDetectUtils import AutoDetectUtils
 from .JGIMetadata import read_metadata_for, translate_for_importer
 from .auth2Client import KBaseAuth2
 from .globus import assert_globusid_exists, is_globusid
 from .metadata import some_metadata, dir_info, add_upa, similar
 from .utils import Path, run_command, AclManager
 
 routes = web.RouteTableDef()
-VERSION = '1.1.7'
+VERSION = "1.1.7"
 
 
-@routes.get('/add-acl-concierge')
+@routes.post("/importer_mappings/{query:.*}")
+async def importer_mappings(request: web.Request) -> web.json_response:
+    """
+    Return a dictionary with two lists: apps and mappings
+    Apps are a list of importers
+    Mappings are a list of mapping between passed in files, and available apps
+    :param request: contains a list of files e.g. ['file1.txt','file2.fasta']
+    """
+    file_list = parse_qs(await request.text()).get("file_list", [])
```

Comment:
Should it raise a 400 error if there are no valid file names in the list?

Reply:
@ialarmedalien No it shouldn't, it should just return no match found